Localization/en-US/faq.po

```diff
@@ -0,0 +1,17 @@
+# Schema-Version: 1.0
+# URL: https://foldingathome.org/support/faq/
+# Source-Language: en-US
+# Source-Version: 1.0
+# Source-Last-Modified: 20200705
+# Target-Language:
+# Target-Version:
+# Target-Last_Modified:
+
+msgid "FAQ"
+msgstr ""
+
+msgid "**You?ll find a large library of frequently asked questions about all things connected to Folding@home.**"
```

Comment:
```suggestion
msgid "You'll find a large library of frequently asked questions about all things connected to Folding@home."
```

Looks like there is a little problem with special characters. That should probably be `You'll` ;)

Also...as this entire statement is bold, maybe we don't extract it with Markdown, but assume wherever the string goes it will be made bold by the layout?